docs/environment_variables.md

```diff
@@ -0,0 +1,47 @@
+Environment Variables
+=====================
+
+The HTTPX library can be configured via environment variables.
+Here are a list of environment variables that HTTPX recognizes
+and what function they serve:
+
+HTTPX_DEBUG
```

Comment:
```suggestion
`HTTPX_DEBUG`
```

Reply:
Or really? I looked at the "Developer Interface" page and e.g. `Client` is rendered in a monospace font. Perhaps it's the use of `===` / `---` instead of `#` and `##` for headers?

Reply:
I checked, and regardless of the header markup style (which I think we could/should convert to using shebangs since that's what we use everywhere else?) I do get monospace when using code fences:

![Screenshot 2019-08-26 at 23 43 38](https://user-images.githubusercontent.com/15911462/63725674-5da22f80-c85b-11e9-91f8-c7391ee0303b.png)